### PR TITLE
Add com.simulide.simulide

### PR DESCRIPTION
**Pull request by giann1s**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I have [built][build] and tested the submission locally.
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines][app-id] are followed.
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission
[app-id]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id

### CODE REVIEW

File: com.simulide.simulide.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=home
+modules:
+  - name: simulide
+    buildsystem: qmake
+    subdir: build_XX
+    build-commands:
```
Comment (hfiguiere) [inline com.simulide.simulide.yml:15]: this should be in the post-install if it is not in qmake install rule.

File: com.simulide.simulide.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      - install -Dm755 -t /app/bin/ ./executables/SimulIDE_1.0.0-SR2/simulide
+      - install -d /app/share/simulide/
+      - cp -r ./executables/SimulIDE_1.0.0-SR2/data /app/share/simulide
+      - cp -r ./executables/SimulIDE_1.0.0-SR2/examples /app/share/simulide
+      - install -d /app/share/simulide/translations/
+      - cp ../resources/translations/*.qm /app/share/simulide/translations/
+    sources:
+      - type: bzr
+        url: lp:simulide/1.0.0
```
Comment (hfiguiere) [inline com.simulide.simulide.yml:24]: you need a revision.

Comment (hfiguiere) [reply com.simulide.simulide.yml:24]: also is this a valid repo URL?

**Comment by hfiguiere** [comment]
bot, build com.simulide.simulide

File: com.simulide.simulide.yml (hunk 2 above)
Comment (giann1s) [reply com.simulide.simulide.yml]: The developer added the xml file recently, so it's not yet on the zip. I included it temporarily on this repository and I will change it back when the next version is released.

**Comment by giann1s** [comment]
bot, build com.simulide.simulide

File: com.simulide.simulide.yml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+      - install -d /app/share/simulide/
+      - cp -r ./executables/SimulIDE_1.0.0-SR2/data /app/share/simulide
+      - cp -r ./executables/SimulIDE_1.0.0-SR2/examples /app/share/simulide
+      - install -d /app/share/simulide/translations/
+      - cp ../resources/translations/*.qm /app/share/simulide/translations/
+      - install -Dm644 ../resources/icons/simulide.png -t /app/share/icons/hicolor/256x256/apps/
+      - install -D ../resources/simulide.desktop /app/share/applications/com.simulide.simulide.desktop
+      - install -D ../com.simulide.simulide.metainfo.xml /app/share/metainfo/com.simulide.simulide.metainfo.xml
+
+rename-icon: "simulide"
```
Comment (bbhtt) [inline com.simulide.simulide.yml:32]: This should be on top before finish-args.

File: com.simulide.simulide.metainfo.xml
Comment (bbhtt) [inline com.simulide.simulide.metainfo.xml:1]: THis should be in the upstream repo of the app.

**Comment by giann1s** [comment]
bot, build com.simulide.simulide

**Pull request closed without merging**